docs/MESSAGING.md

```diff
@@ -19,10 +19,30 @@ Open `app/App_Resources/iOS/Info.plist` and add this to the bottom:
 ```xml
 <key>UIBackgroundModes</key>
 <array>
-	<string>remote-notification</string>
+    <string>remote-notification</string>
 </array>
 ```
 
+For Android you will not get the title and body if the notification was received while the application was in background, but you will get the *data* payload.
+Add the following services in the `app/App_Resources/Android/AndroidManifest.xml` to enable advanced FCM messaging:
+```
+<manifest ... >
```

Comment:
I think it is ok for the plugins to require some modifications to be done by hand, as long as these modifications can be kept in source control and are well documented.

If the changes happen to be in platforms/* then it would be best if there is an automated way to apply them (using hooks?) so the app can roundtrip source control and continuous integration.